backend/src/server.ts

```diff
@@ -1,7 +1,6 @@
-import 'dotenv/config';
 import { app } from './app';
 
-const PORT = process.env.PORT || 3000;
+const PORT = 3000;
```

Comment:
Why did you hard code this and don't use a env-var?

Reply:
I wasn't sure where to import the inital `import 'dotenv/config';`, so I put it into db.ts and hard coded the port. But you are right, it is not the proper way. Now I added the import to both entry points: index.ts for local developement and api.ts for the Netlify serverless function. That works and is the proper way, I assume.

Reply:
I am not sure what the the right way is here but i would assume it doesn't matter where you import the  `import 'dotenv/config'` or rather i would import it in the files you need it, no? So i would have imported it in the db file as well as in the server file. Or does this not work?